src/VisualStudio/Core/Next/Remote/JsonRpcSession.cs

```diff
@@ -65,43 +65,29 @@ internal class JsonRpcSession : RemoteHostClient.Session
 
         private async Task InitializeAsync()
         {
-            CancellationToken.ThrowIfCancellationRequested();
-
-            try
-            {
-                // all roslyn remote service must based on ServiceHubServiceBase which implements Initialize method
-                await _snapshotClient.InvokeAsync(WellKnownServiceHubServices.ServiceHubServiceBase_Initialize, _currentSessionId, PinnedScope.SolutionChecksum.ToArray()).ConfigureAwait(false);
-                await _serviceClient.InvokeAsync(WellKnownServiceHubServices.ServiceHubServiceBase_Initialize, _currentSessionId, PinnedScope.SolutionChecksum.ToArray()).ConfigureAwait(false);
-            }
-            catch (ObjectDisposedException)
```

Comment:
these try/catch is not needed. looks like haven't been cleaned up when try/catch is moved into InvokeAsync